pyatlas/pyatlas/atlas.py

```diff
@@ -0,0 +1,1239 @@
+
+    def _load_relationIdentifiers(self):
+        zip_entry_data = _read_zipentry(self.atlas_file, self._FIELD_RELATION_IDENTIFIERS)
+        self.atlas.relationIdentifiers = autogen.ProtoLongArray_pb2.ProtoLongArray()
+        self.atlas.relationIdentifiers.ParseFromString(zip_entry_data)
+
+    def _load_relationIdentifierToRelationArrayIndex(self):
+        zip_entry_data = _read_zipentry(self.atlas_file,
+                                        self._FIELD_RELATION_IDENTIFIER_TO_RELATION_ARRAY_INDEX)
+        proto_map = autogen.ProtoLongToLongMap_pb2.ProtoLongToLongMap()
+        proto_map.ParseFromString(zip_entry_data)
+        self.atlas.relationIdentifierToRelationArrayIndex = _get_dict_from_longtolongmap(proto_map)
+
+    def _load_relationMemberTypes(self):
+        zip_entry_data = _read_zipentry(self.atlas_file, self._FIELD_RELATION_MEMBER_TYPES)
+        self.atlas.relationMemberTypes = autogen.ProtoByteArrayOfArrays_pb2.ProtoByteArrayOfArrays()
+        self.atlas.relationMemberTypes.ParseFromString(zip_entry_data)
+
+    def _load_relationMemberIndices(self):
+        zip_entry_data = _read_zipentry(self.atlas_file, self._FIELD_RELATION_MEMBER_INDICES)
+        self.atlas.relationMemberIndices = autogen.ProtoLongArrayOfArrays_pb2.ProtoLongArrayOfArrays(
+        )
+        self.atlas.relationMemberIndices.ParseFromString(zip_entry_data)
+
+    def _load_relationMemberRoles(self):
+        zip_entry_data = _read_zipentry(self.atlas_file, self._FIELD_RELATION_MEMBER_ROLES)
+        self.atlas.relationMemberRoles = autogen.ProtoIntegerArrayOfArrays_pb2.ProtoIntegerArrayOfArrays(
+        )
+        self.atlas.relationMemberRoles.ParseFromString(zip_entry_data)
+
+    def _load_relationTags(self):
+        zip_entry_data = _read_zipentry(self.atlas_file, self._FIELD_RELATION_TAGS)
+        proto_relation_tags = autogen.ProtoPackedTagStore_pb2.ProtoPackedTagStore()
+        proto_relation_tags.ParseFromString(zip_entry_data)
+        self.atlas.relationTags = _get_packed_tag_store_from_proto(proto_relation_tags)
+
+    def _load_relationIndexToRelationIndices(self):
+        zip_entry_data = _read_zipentry(self.atlas_file,
+                                        self._FIELD_RELATION_INDEX_TO_RELATION_INDICES)
+        proto_multimap = autogen.ProtoLongToLongMultiMap_pb2.ProtoLongToLongMultiMap()
+        proto_multimap.ParseFromString(zip_entry_data)
+        self.atlas.relationIndexToRelationIndices = _get_dict_from_protolongtolongmultimap(
+            proto_multimap)
+
+    def _load_field(self, field_name):
+        if field_name not in self._FIELD_NAMES_TO_LOAD_METHODS:
+            raise KeyError('unrecognized field {}'.format(field_name))
+
+        # reflection code to get the appropriate load method for the field we are loading
+        method_name = self._FIELD_NAMES_TO_LOAD_METHODS[field_name]
+        load_method_to_call = getattr(self, method_name)
+        load_method_to_call()
+
+
+class _PackedTagStore(object):
```

Comment:
Are those included in the same file to make them "private"?

Reply:
It's not the fact that they are included in the same package, rather the leading `_` character is what makes them private. You can see in the `geometry` module that there are many classes defined there, none of which are private.

The reason I placed `_PackedTagStore` and `_IntegerDictionary` in the `atlas` module is because they are only called from within that module, and they don't need to be exposed to the user for import or documentation.